hw/ip/otbn/doc/_index.md

```diff
@@ -426,11 +426,73 @@ Rough expected process:
 
 ## Error conditions
 
-<div class="bd-callout bd-callout-warning">
-  <h5>Note</h5>
-
-  To be filled in as we create the implementation.
-</div>
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Code</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>No Error</td>
+      <td>0x00</td>
+      <td>Execution was successful</td>
+    </tr>
+    <tr>
+      <td>Bad Data Address</td>
+      <td>0x01</td>
+      <td>
+      A Dmem read or write occurred with an out of bounds or unaligned address
+      </td>
+    </tr>
+    <tr>
+      <td>Bad Instruction Address</td>
+      <td>0x02</td>
+      <td>
+      An instruction fetch occurred with an out of bounds or unaligned address
+      </td>
+    </tr>
+    <tr>
+      <td>Call Stack over/underflow</td>
+      <td>0x03</td>
+      <td>
+      A push to the call stack when it was full or a pop from the call stack
+      when it was empty was attempted
+      </td>
+    </tr>
+    <tr>
+      <td>Illegal instruction</td>
+      <td>0x04</td>
+      <td><ul>
+        <li>An instruction being excuted had an invalid encoding</li>
+        <li>A CSR or WSR access occurred to an invalid CSR or WSR</li>
```

Comment:
I think this doesn't need the duplication of "CSR or WSR". Maybe just "An access occurred for an invalid CSR or WSR"? (And I don't think accesses occur *to* things; "for" is maybe better?)